backend/uclapi/oauth/scoping.py

```diff
@@ -16,7 +16,9 @@
 
 
 class Scopes:
-    SCOPE_MAP = {}
+    SCOPE_MAP = {
+        "timetable": (1, "Personal Timetable"),

```

Comment:
Can be both, but realistically you should start from 0 as `2 << 0` is `1` so you would have missed out a scope.

Reply:
I actually suggested 1, because 0 is for anything to do with room bookings (it's earmarked internally and some of the tests I wrote a while ago expect that)

Reply:
it could be any number up to 63 though

Reply:
Can we be explicit and define it in the `SCOPE_MAP`?

Reply:
No, because otherwise it'll render to clients that there's private room bookings data available (which there isn't). It's there in case we in the future decide to offer all room bookings a particular person has made.

Reply:
(which would be a nice feature, but it's not useful for the majority of students because the only students who can book rooms are society presidents)